gremlin-go/driver/protocol.go

```diff
@@ -116,19 +116,19 @@ func (protocol *gremlinServerWSProtocol) responseHandler(resultSets *synchronize
 	if statusCode == http.StatusNoContent {
 		resultSets.load(responseIDString).addResult(&Result{make([]interface{}, 0)})
 		resultSets.load(responseIDString).Close()
-		log.logger.Log(Info, "No content.")
+		log.logStr(Info, "No content.")
 		log.logf(Info, readComplete, responseIDString)
 	} else if statusCode == http.StatusOK {
 		// Add data and status attributes to the ResultSet.
 		resultSets.load(responseIDString).addResult(&Result{data})
 		resultSets.load(responseIDString).setStatusAttributes(response.responseStatus.attributes)
 		resultSets.load(responseIDString).Close()
-		log.logger.Logf(Info, "OK %v===>%v", response.responseStatus, data)
+		log.logStrf(Info, "OK %v===>%v", response.responseStatus, data)
```

Comment:
I think that No content. OK ===> and Partial ===> should all be removed.

Reply:
removed